packages/gatsby-plugin-mdx/loaders/mdx-loader.js

```diff
@@ -182,12 +182,9 @@ export const _frontmatter = ${JSON.stringify(data)};`
   // check needs to happen first.
   if (!hasDefaultExport(content, options) && !!defaultLayout) {
     debug(`inserting default layout`, defaultLayout)
-    const { content: contentWithoutFrontmatter, matter } = grayMatter(
-      content,
-      options
-    )
+    const { content: contentWithoutFrontmatter } = grayMatter(content, options)
 
-    code = `${matter ? matter : ``}
```

Comment:
When `grayMatter` was not passed options prior to this change, `matter` would always be `undefined`, resulting in no frontmatter rendered to the page. This changes removes `matter` entirely from the output node raw body.

Unsure if this is the correct fix since it's unclear to me why `matter` was included here in the first place (maybe it is stripped in later code paths?).

Reply:
I tracked first introduction of it to https://github.com/ChristopherBiscardi/gatsby-mdx/pull/329 - there is 0 context on it - to me it seem more like debugging code than anything else - the `code` is in the MDX format, so it would just output any content put there as content which doesn;t make much sense (to me anyway).

Especially with fact that `matter` in returned object means:

> file.matter {String}: the raw, un-parsed front-matter string